RetailCoder.VBE/Root/RubberduckModule.cs

```diff
@@ -158,8 +157,18 @@ public override void Load()
             ConfigureProjectExplorerContextMenu();
             
             BindWindowsHooks();
+        }
 
+        private IEnumerable<Assembly> FindPlugins()
+        {
+            var basePath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+            yield return Assembly.LoadFile(Path.Combine(basePath, "Rubberduck.Inspections.dll"));
```

Comment:
So `Rubberduck.Inspections.dll` is essentially a plug-in... but since it's the "core inspections package" we're loading it from the main assembly path.